stable/gcp-night-king/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "nightking.name" -}}
```

Comment:
The template namespace should be the same a the chart name: `gcp-night-king`.

Reply:
right. fixed.